lib/child_process.js

```diff
@@ -89,7 +92,9 @@ exports.fork = function(modulePath /*, args, options*/) {
     options.stdio = options.silent ? stdioStringToArray('pipe') :
       stdioStringToArray('inherit');
   } else if (options.stdio.indexOf('ipc') === -1) {
-    throw new TypeError('Forked processes must have an IPC channel');
+    throw new errors.TypeError('ERR_INVALID_OPT_VALUE',
+                               'options.stdio',
+                               options.stdio);
```

Comment:
I think a more specific error is better for this.
e.g. `new errors.Error('ERR_CHILD_PROCESS_IPC_REQUIRED')` with the original error message in tact.